src/api/rooms/getRoomDetail.ts

```
@@ -42,8 +42,19 @@ export const getRoomDetail = async (roomId: number): Promise<RoomDetailResponse>
   try {
     const response = await apiClient.get<RoomDetailResponse>(`/rooms/${roomId}/recruiting`);
     return response.data;
-  } catch (error) {
+  } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)